modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json

```diff
@@ -16,11 +16,13 @@
 	"add-activation-key": "Add Activation Key",
 	"add-another-admin": "Add Another Admin",
 	"add-at-least-one-user-s-email-to-send-an-invitation": "Add at least one user's email to send an invitation.",
+	"add-contacts-to-be-notified-in-the-even-of-a": "Add contacts to be notified in the even of a",
 	"add-incident-report-contact": "Add Incident Report Contact",
 	"add-more-members": "Add More Members",
 	"add-one-ip-addresses-per-line-ipv6-addresses-are-not-supported": "Add one IP address per line. IPv6 addresses are not supported.",
 	"add-one-mac-addresses-per-line": "Add one MAC address per line.",
 	"add-one-mac-addresses-per-line-if-available-as-a-static-value": "Add one MAC address per line, if available as a static value.",
+	"add-or-remove-contacts-to-be-notified-in-the-even-of-a": "Add or remove contacts to be notified in the even of a",
```

Comment:
same here, is this supposed to say `event` instead of `even`?

Reply:
Thanks Kyle, it would be better, but in order not to stop "ci:forward" I'll fix it in the next subtask!